aicssegmentation/tests/test_WorkflowEngine.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from unittest.mock import MagicMock, create_autospec
+import numpy as np
+from aicssegmentation.structure_wrapper.WorkflowEngine import WorkflowEngine
+
+class TestWorkflowEngine:
+    def setup_method(self):
+        self.fake_image = np.zeros([100,100])
+        self.engine = WorkflowEngine("sec61b", self.fake_image)
+
+    def test_get_next_step(self):
+        assert self.engine.get_next_step().name == "Intensity Normalization"
+        self.engine.execute_next()
+        assert self.engine.get_next_step().name == "Edge Preserving Smoothing"
+        self.engine = WorkflowEngine("sec61b", self.fake_image)
+
+    def test_execute_next(self):
+        assert self.engine.currentStep == 0
+        assert self.engine.steps[0].result is None
+        self.engine.execute_next()
```

Comment:
This might be a matter of preference but I find it a bit confusing that we're calling a function called `execute_next()` but we're really executing `currentStep` not `nextStep`. Not sure what the best way to fix this would be, maybe renaming `execute_next()` as `execute_step()`? Or renaming `currentStep` as `nextStep`?

(Also in ln 20 of WorkflowEngine.py your comment says "next step to execute" but the property name is `self.currentStep` 😛)

Reply:
Yeah this is a good point! Definitely a little confusing. For clarity I'm renaming currentStep to nextStep, since that's what that value represents.